server/aws-lsp-partiql/src/server/language-server.ts

```diff
@@ -0,0 +1,72 @@
+import { type DidChangeTextDocumentParams, type Server } from '@aws/language-server-runtimes/server-interface'
+import { TextDocumentSyncKind, type DidOpenTextDocumentParams } from 'vscode-languageserver'
+import { TextDocument } from 'vscode-languageserver-textdocument'
```

Comment:
nit: `TextDocument` interface is available through Runtimes `@aws/language-server-runtimes/server-interface`, so direct dependency on `vscode-languageserver-textdocument` in code.

`TextDocumentSyncKind` and `DidOpenTextDocumentParams` should be also available, so direct dependency on VSCode LSP packages can be minimized even more.